site/content/en/docs/contributing/development-environment.md

```diff
@@ -27,6 +27,28 @@ description: 'Installing a development environment for different operating syste
   brew install git python pyenv redis curl openssl node sqlite3 geos
   ```
 
+  Arch Linux
+  ```bash
+  # Update the system and AUR (you can use any other AUR helper of choice) first:
+  sudo pacman -Syyu
+  pikaur -Syu
+  ```
+
+  ```bash
+  # Install the required dependencies:
+  sudo pacman -S base-devel curl git redis cmake gcc python python-pip tk libldap libsasl pkgconf ffmpeg geos openldap python-lda
+  ```
+
+  ```bash
+  # Since migration only works with Python<=3.9, because the Iterable class has been removed from collections in Python 3.10, install Python<=3.9 if you don’t have it:
```

Comment:
Please update the wording in the comment from the point that Python 3.9 is expected and the only supported by the CVAT server.

Reply:
Sure, specified the only supported Python version in the comment.